.github/workflows/build_test_deploy.yml

```
@@ -41,16 +41,16 @@
         run: yarn check-cyclic-dependencies
       - name: Run frontend tests
         run: yarn run vitest run --coverage.enabled true
-      # - name: Download Coverage Artifacts 
-      #   uses: actions/download-artifact@v4
-      #   with:
-      #     name: vitest-coverage-master
-      #     path: coverage
+      - name: Download Coverage Artifacts 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)